package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Convert MathML documents to SVG images",
   "keywords": [],
   "dependencies": {
-    "htmlparser2": "~3.7.1",
-    "pdfkit": "0.6.1",
-    "ent": "~2.0.0"
+    "htmlparser2": "^3.9.1",
+    "pdfkit": "^0.8",
```

Comment:
This was intentionally pinned to a specific version.  Although it can be updated, it should not have the `^` because we are using an internal/non-public API from pdfkit to do font-measurement.  pdfkit would be within their rights to break that in any version.